src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -181,12 +181,13 @@ internal static bool IsLiteral(SyntaxKind kind)
             {
                 case SyntaxKind.IdentifierToken:
                 case SyntaxKind.StringLiteralToken:
+                case SyntaxKind.SingleLineRawStringLiteralToken:
```

Comment:
Do we have tests covering this change (and below `IsAnyToken`)?

Reply:
i can't find any tests covering any of this (before/after my change).

Reply:
Let's add direct test of public APIs modified in this file then.

Reply:
ok.  i added that :)